src/app/modules/group/components/group-manager-list/group-manager-list.component.ts

```diff
@@ -22,15 +22,28 @@ export class GroupManagerListComponent implements OnChanges {
     this.reloadData();
   }
 
+  private convertText(can_manage: string): string {
+    switch(can_manage) {
+      case 'none':
+        return 'Read-only';
+      case 'memberships':
+        return 'Memberships';
+      case 'memberships_and_group':
+        return 'Memberships and group';
+      default:
+        return '';
```

Comment:
Just one last thing, I don't exactly know what to return if the `can_manage` value is not one of the one of the desired value.

Reply:
Hum indeed, this is not really correct. You cannot really consider that `can_manage` is a string as what is returned by the service is a union type (you "downcast" it, you lose information). You should probably define an exported type in the service for this union type, use this type here, and you will not have to handle any default case.